Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM mono:3.12.1 as builder
+FROM mono as builder
```

Comment:
I think `FROM mono:latest` would be more explicit, although I'm always hesitant to use "latest" for anything. How about `FROM mono:6.8`, which would  pick up 6.8* according to https://hub.docker.com/_/mono/

I don't think that a multi-stage build is required at all now, since both this Dockerfile and the upstream mono image are both Debian Stretch?